schematic/store/synapse.py

```diff
@@ -1724,7 +1818,6 @@ def getDatasetProject(self, datasetId: str) -> str:
             str: The Synapse ID for the parent project.
         """
 
-        self._query_fileview()
```

Comment:
Removing this will potentially cause issues for table operations when in use, and will likely cause issues with tests covering the function of table operations

Reply:
After I merged the latest develop to this branch, I then tested by running TestTableOperations locally. Even though the first two times, I got a strange lookup error, the error seemed to be resolved when I tried the second and third time.